### PR TITLE
cmd: fix megacheck warnings

### PR DESCRIPTION
**Pull request by egonelbre**
megacheck warnings left:

```
cmd\utils\flags.go:1106:63: event.TypeMux is deprecated: use Feed  (SA1019)
```

### CODE REVIEW

File: cmd/faucet/faucet.go [hunk 1]
```diff
@@ -471,7 +471,7 @@ func (f *faucet) apiHandler(conn *websocket.Conn) {
 
 		// Send an error if too frequent funding, othewise a success
 		if !fund {
-			websocket.JSON.Send(conn, map[string]string{"error": fmt.Sprintf("%s left until next allowance", common.PrettyDuration(timeout.Sub(time.Now())))})
+			websocket.JSON.Send(conn, map[string]string{"error": fmt.Sprintf("%s left until next allowance", common.PrettyDuration(time.Until(timeout)))})
```
Comment (karalabe) [inline cmd/faucet/faucet.go]: This fix requires Go 1.8. We do not yet want to drop builds with Go 1.7.x, so please revert this change.

File: cmd/puppeth/wizard.go [hunk 2]
```diff
@@ -160,27 +160,6 @@ func (w *wizard) readDefaultInt(def int) int {
...
-			continue
-		}
-		val, err := strconv.ParseFloat(strings.TrimSpace(text), 64)
-		if err != nil {
-			log.Error("Invalid input, expected float", "err", err)
-			continue
-		}
-		return val
-	}
-}
```
Comment (karalabe) [inline cmd/puppeth/wizard.go]: I'm unsure about this change. Although it's unused, it might be worthwhile to have it in there if someone needs it for the future. What's your take on it?

File: cmd/faucet/faucet.go (hunk 1 above)
Comment (egonelbre) [inline cmd/faucet/faucet.go]: Got it, I think there was a few other places, where I replaced it.

File: cmd/puppeth/wizard.go (hunk 2 above)
Comment (egonelbre) [inline cmd/puppeth/wizard.go]: I guess there are three other alternatives:

* just show the megacheck warning: this is somewhat counter-productive, eventually the megacheck log should be empty, to spot the errors more easily
* comment out the code: easy to understand, no problem with uncommenting
* use `var _ = (*wizard)(nil).readFloat` - to mark it as needed

Commenting out seems the most reasonable alternative here. 

PS: There might be even better way of writing these read functions, with less repetition, but probably not good idea in this PR.
